music.go

```diff
@@ -0,0 +1,53 @@
+package pctk
+
+import (
+	"log"
+	"os"
+	"unsafe"
+
+	rl "github.com/gen2brain/raylib-go/raylib"
+)
+
+// Music is music and we love it!
+type Music = rl.Music
+
+func (a *App) LoadMusic(fileName string) {
```

Comment:
I think the music must be considered a resource, and hence `LoadMusic()` should receive a `ResourceLocator` as argument and load the music object from its resource loader. 

Please note that any game asset must be obtained using the `ResourceLoader` interface passed to the app in its construction.

Reply:
Done